recipes/arrow/all/conanfile.py

```diff
@@ -232,7 +232,7 @@ def requirements(self):
         if self.options.with_json:
             self.requires("rapidjson/1.1.0")
         if self._with_llvm():
-            raise ConanInvalidConfiguration("CCI has no llvm recipe (yet)")
+            self.requires("llvm-core/[>=11.1.0]")
```

Comment:
Do not use version ranges in CCI. It might change packageID and consumers will complain about _missing packages_.